frontend/www/js/omegaup/components/arena/NavbarProblems.vue

```diff
@@ -98,7 +106,11 @@ export default class ArenaNavbarProblems extends Vue {
   }
 
   getMaxScoreForProblem(problem: types.NavbarProblemsetProblem): string {
-    return `(${problem.bestScore.toFixed(
+    const bestScore =
+      problem.myBestScore !== undefined
+        ? problem.myBestScore
+        : problem.bestScore;
```

Comment:
¿Qué crees? en javascript también existe el [`Nullish coalesing operator`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing), así que esta expresión la puedes simplificar de la siguiente forma:
```suggestion
    const bestScore = problem.myBestScore ?? problem.bestScore;
```